go/minterm/minterm.go

```diff
@@ -78,6 +78,9 @@ func (m *MinTerm) fdIn() int { return int(m.termIn.Fd()) }
 func (m *MinTerm) getNewTerminal(prompt string) (*terminal.Terminal, error) {
 	term := terminal.NewTerminal(m.getReadWriter(), prompt)
 	a, b := m.Size()
+	if a < 80 {
+		a, b = 80, 24
+	}
```

Comment:
is there a reason not to do height independently?

Reply:
I was thinking someone could realistically have a short terminal, say in a tiling wm.

Reply:
height probably doesn't matter anyway, but we likely aren't calculating the right thing here for example when someone resizes their terminal to 70x60 for some reason